GALLERY.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+title: Gallery
+nav_order: 1
+---
+
+# Gallery
+
+Examples renderings with their associated command lines.
+Images and videos displayed below use public datasets, you can download them [here](https://drive.google.com/file/d/1iRh0OeJjMjjaBDLG6b_iJSkC_Jt_YQuo/view?usp=sharing).
+
+<video src='https://user-images.githubusercontent.com/3129530/194735274-395839e9-e2cb-4a47-83b9-5c6987640032.webm' autoplay="autoplay" loop="loop" width="700"></video>
```

Comment:
About videos, they are looking great in jekyll but in github they use the github player, which does not lets us use autoplay. I think this is fine. The alternative is to use .gif everywhere, but the quality is just not the same.

Reply:
@Meakk

Reply:
Yes, videos are better. I don't care if the autoplay is disabled on github.

Reply:
since README is not used anymore in jekyll, i've fixed that in HOME.md

Reply:
Using a video in README.md now, looks not great in github.com imo, let me know what you think. we can:

1. Duplicate README.md to have different document between jekyll and github, ok for a single file imo
2. Use gif, nicely integrated everywhere but not sure beatiful
3. Use video, badly integrated in github.com, but looks superb on jekyll

let me know